data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,462 @@
+			`	"This isn't a base; it's a colony."`
+			`	You notice capital ships heading your way and about two dozen more preparing to leave the planet. It's time to go.`
+	on complete
+		payment 775000
+	npc
+		government "Pirate (Devil-Run Gang)"
+		system "Devil-Hide"
+		fleet "Devil-Run Warships" 2
+		personality nemesis harvests plunders disables
+
+mission "Hai Rescue [A09] Slaver Ships"
+	landing
+	name "Missing Researchers"
+	description "Report back to Teeneep on <destination>. Payment is <payment>."
+	source "Earth"
+	destination "Darkwaste"
+	to offer
+		has "Hai Rescue [A08] Scanner: done"
+	on offer
+		conversation
+			`The Earth authorities give you an unusual landing vector that leads to a secret docking bay below ground in Argentina. You land and a Navy cadet meets you at your ship with an urgent message, "Republic Intelligence and a Navy Marshal want to see you as soon as possible. They have news about your microtransceiver transmissions. Please follow me."`
+			`	Giti adds, "I'll take the scanners and data to some researchers and let you know when I have more information. Please go with the cadet. It sounds like you'll finally get some answers."`
+			`	The cadet takes you to an elevator just outside your ship, down more than three hundred floors to a large meeting room. Within, you see two men and a woman in formal dress. One man has a Republic Intelligence uniform with the name, Agent Kopkow. Another wears a navy full dress uniform that you don't immediately recognize. After a moment of thought, you realize he is a Marshal, the second-highest rank in the Navy, answering only to the Grand Marshal. His uniform is is labeled, Adurath Kaine. The woman is a Navy captain wearing the prominent Parliamentary Medal of Valor. Her name label says, "Captain Anna Wallace."`
+			choice
+				`	(Speak to the agent and marshal.)`
+					goto join
+				`	(Ruminate first.)`
+			`	You've had few dealings with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle. Murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament.`
+			`	There are only five Navy Marshals, and one Grand Marshal; this man's high rank means he reports parliament regularly. People this high in the navy tend to deal more with political issues, rather than combat or strategy, so you wonder why they picked him to talk to you.`
```

Comment:
```suggestion
			`	There are only five Navy Marshals, and one Grand Marshal; this man's high rank means he reports to Parliament regularly. People this high in the Navy tend to deal more with political issues rather than combat or strategy, so you wonder why they picked him to talk to you.`
```
Is `Parliament` capitalized? Dunno, but I thought it might need to be. I can edit the suggestion to not include it if that's wrong